build_scripts/build.sh

```diff
@@ -48,7 +48,7 @@ if [ "$ENABLE_DX" == "1" ]; then
 	run_buildscripts_for "$(arch)/dx"
 fi
 
-if [ "$ENABLE_GDX" == "1" ] ; then
+if [ "$ENABLE_GDX" == "1" ]; then
 	# We explicitly only support x86 on nvidia (unless they update it?)
 	copy_systemfiles_for "gdx"
 	copy_systemfiles_for "x86_64-gdx"
```

Comment:
```suggestion
	copy_systemfiles_for "$(arch)-gdx"
```
(plus the line below, which I can't suggest on this PR)
For if we do eventually get Nvidia ARM :)

Reply:
Id like to do this change in another PR, this one is meant to be "stylistic" (minor) changes only. Im trying to make it so the nvidia driver uses the negativo kmod on another PR that ill do soon